src/cliques/cliquer/cliquer.h

```diff
@@ -28,7 +28,7 @@ struct _clique_options {
 				 clique_options *);
 	FILE *output;
 
-	boolean (*user_function)(set_t,graph_t *,clique_options *);
```

Comment:
This one I'm not completely sure of. This changes the interface, but we were putting errors in a boolean.

Reply:
I think Cliquer's `user_function` is a private API so we can do whatever we want with it.

Reply:
As far as igraph is concerned it's private, yes. Diverging from Cliquer is a bit uncomfortable, but the Cliquer author did say that he is not planning new releases.